pkg/network/protocols/http/ebpf_ssl.go

```diff
@@ -388,7 +385,7 @@ func addHooks(m *errtelemetry.Manager, probes []manager.ProbesSelector) func(str
 
 				newProbe := &manager.Probe{
 					ProbeIdentificationPair: identifier,
-					BinaryPath:              libPath,
+					BinaryPath:              root + path,
```

Comment:
```suggestion
					BinaryPath:              filepath.Join(root, path),
```

Reply:
we control root and path, and the kernel will solve the path himself when the file is used
filepath.Join() is nice but why spending cpu cleanup-ing a special path ?